### ISSUE TITLE
Nerf Militarist & Pacifist support gains from combat, buff Religious

### ISSUE THREAD

**Issue by Tat011** [captaincobbs/ESG-Mod#1484]
It's no secret that Militarist & Pacifist get way over represented in elections and i'm pretty sure that fighting is to blame, since EVERY fight boosts one party or the other, so i'm proposing support generated from all fighting get significantly nerfed.
On the other hand it's impossible for Religious to win an election without Religious pops regardless of what you do, tbh i'm not quite sure how to solve this one besides moving the "take quest" minor action from Pacifist to Religious.

**Comment by captaincobbs** [comment]
If you're willing to go through and balance this I'd say you're good to go ahead and work on this.

**Comment by wakystuf** [comment]
Yeah I would agree. Do the variables exist in the system to reduce pacifist and militarist support from attacking/defending? If not, then we could just remove the pacifist one.
Then as you correctly point out, we should probably find a way to increase religious, but I'm not sure what would be best.

**Comment by captaincobbs** [comment]
> Yeah I would agree. Do the variables exist in the system to reduce pacifist and militarist support from attacking/defending? If not, then we could just remove the pacifist one. Then as you correctly point out, we should probably find a way to increase religious, but I'm not sure what would be best.

They do exist, they exist as political events you can mess with the weights of. There are a LOT of disabled ones we could affect. When I last went through it I made religious be primarily generated through assigning a hero and having high approval.

**Comment by Tat011** [comment]
While looking to move ideologies to religious i noticed that A LOT of techs dont match their unlocks with their ideology so this fixes that, ultimately this does buff religious but also significantly nerfs ecologist and buffs scientist which we'll have to account for.
Tech ideology change proposal:
Military:
-Behemoth Weaponization: Militarist > Religious

-Both of the 3 Juggernaut/Obliterator techs: Militarist > Religious

Economy and Trade:
-Atmospheric Filtration: Pacifist > Ecologist

-Impactless Site: Industrialist > Pacifist 
-Mission Flexibility: Pacifist > Religious

-Ultratough Materials: Pacifist > Industrialist

-Xeno NLP: Industrialist > Pacifist

Science and Exploration:
-Xenobiology: Ecologist > Scientist

-Eucariotic Sap: Scientist > Ecologist
-Baryonic Shielding: Ecologist > Scientist

-Wave Function Control: Ecologist > Scientist
-Compact Warp Methods: Ecologist > Scientist
-Graviton Research: Ecologist > Scientist
-Autonomous Materials: Ecologist > Scientist
-Behemoth Basic Climatology: Scientist > Ecologist

-Adaptive Colonies: Religious > Ecologist
-Custom Nucleotides: Scientist > Ecologist
-N-Dimensional Topologies: Ecologist > Scientist
-Low Kelvin Sciences: Ecologist > Scientist
-Ultra-Deep Habitats: Ecologist > Scientist

-Ontological Rev-Eng: Ecologist > Scientist
-Advanced Non-Baryonics: Ecologist > Scientist

Empire Development:
-Adaptive Bureaucracies: Pacifist > Religious

-Citadel Blueprints: Industrialist > Religious

-Chlorophyll Chemistry: Religious > Ecologist

**Comment by wakystuf** [comment]
Can you clarify what you mean when you say that the techs don't match their ideology? In general, transferring more support into Religious is probably good (those are rare), but transferring away from Ecologist into Scientist is probably bad (scientist already fairly common).

**Comment by Tat011** [comment]
Lots of techs will give you improvements/bonus of a specific FIDSI then have the ideology associated with completely separate FIDSI, like for Example Impactless Sites being Industrialist despite being the follow up to a pacifist tech and giving you a dust building + hero market.
You can take a look yourself i recently added them to the wiki: https://endless-space-2.fandom.com/wiki/Technology

Yes this would likely make scientists too strong but the change can be accomodated by making the science support from researching any tech weaker or something like that, not sure about ecologists but the problem here is that most food/pop techs are tier 5 for some reason lol.

**Comment by captaincobbs** [comment]
This happened in vanilla, so ESG doesn't need to do it
